### ISSUE TITLE
Globals / Atomic Elements

### ISSUE THREAD

**Issue by rockymountainhigh1943** [basaltinc/knapsack#172]
We need to look at how best to handle atomic elements within Bedrock, because they're important and not going away. It's important that we also look at the distinction between atomic design and component based implementation.

* Atomic Design --> Deals with structural implementation and reusability
* Component Implementation --> Deals with reusable and configurable functionality

Given the importance of keeping both lines separate, not blurred --> I purpose we add a new section for "Atomic Design" and we would bring back the concept of atoms, molecules, and organisms in the "correct" way.
* This new section would look for folders called "atoms", "molecules", and "organisms" and would contain sub-folders within them containing the appropriate twig or mustache files. (think copy/paste from PL and directly into Bedrock)
* This would allow developers and site-builders to truly have a single-source of truth and rapid development. 
* The main idea here is that the atomic elements would eventually be used on their own OR within components.
* From within the page builder --> we would be able to add atoms, molecules, or organisms through a generic pattern placeholder (which would receive special styling) that would contain a dropdown where a user could select the atomic element to insert (e.g. byline, label, heading, etc).
* Important from the above --> No atomic element will ever be adding as a single item to the page builder without using the generic pattern placeholder. In this example, you could not go into the page builder and drag a byline or label over, but would have to use the generic pattern placeholder item.

In Summary --> This feature request solves the problem where developers need to be able to build other things within their design system than just components. Atomic Design is still very important and handles the reusable structural implementation where as component based implementation solves reusable and configurable functionality.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1126343145187461/1136465697085087)

**Comment by joekarasek** [comment]
I believe this has been addressed by 1) custom sections for patterns that allows organization into atomic design (or other system) and 2) templates per component (that allow for the implementation of opinionated sets fo components). I'm proposing that this ticket be closed. 
@EvanLovely Thoughts?
